spec/model/group_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  before :each do
+    @user = User.create(name: 'Tom', email: 'test@gmail.com', password: '1234')
+  end
+
+  it 'Should have the image and title attribute' do
+    group = Group.create!(name: 'Yoga', description: 'THis is a Yoga class', rating: 2, title: 'sport',
+                          image: Rack::Test::UploadedFile.new('spec/support/test_image.png', 'image/png'),
+                          user_id: @user.id)
+    group.image = nil
+    expect(group).to_not be_valid
+  end
```

Comment:
[OPTIONAL] What about valid?

Reply:
Yeah!.. let me implement that.
Thank you :+1: